src/GlobalStyle.js

```diff
@@ -0,0 +1,103 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  /* background-color: #8c94e9; */
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+.center {
+  margin: auto;
+  align-items: center;
+  text-align: center;
+  margin-top: 10%;
+  font-weight: bold;
+  transition: scale 2s;
+  border: 1px solid black;
+  width: 250px;
+  padding: 2%;
+  background-color: #eebdcf;
```

Comment:
Please read about creating global files / themes for colors and font sizes.
Its for defining all colors, fonts, font-size etc in one file and use it throughout the application.
No need to make this change in this PR but add this change from next assignments.